src/types.ts

```diff
@@ -709,3 +709,70 @@ export interface Queue {
     currently_playing: Track | Episode | null
     queue: (Track | Episode)[]
 }
+
```

Comment:
Another solution would be to export the types from this file:

```ts
export type { RecommendationsRequest, RecommendationsRequestRequiredArguments, RecommendationsResponse } from './endpoints/RecommendationsEndpoints'
```

Reply:
True, either is fine with me as long as they can be imported from `'@spotify/web-api-ts-sdk'`

Reply:
@thisisjofrank I made an alternative PR #75 with this method.

Please merge whichever solution you prefer.